app/frontend/resources/assets/js/bootstrap.js

```diff
@@ -45,12 +45,26 @@ if (token) {
 
 window.chosen = require('chosen-js');
 
+/**
+ * simple-lightbox is a lightweight lightbox interface, to make paging through
+ * item photos a little nicer. It will register on elements with 'data-lightbox="show"'
+ */
+
+var SimpleLightbox = require('simple-lightbox');
```

Comment:
this is babel'd code, just do `window.SimpleLightbox = `, and later on use `let`